node/src/chain_spec.rs

```diff
@@ -24,9 +25,60 @@ use sc_chain_spec::{ChainSpecExtension, ChainSpecGroup};
 use sc_service::ChainType;
 use sc_telemetry::TelemetryEndpoints;
 use serde::{Deserialize, Serialize};
-use sp_runtime::Perbill;
+use sha3::{Digest, Keccak256};
+
+use sp_core::{ecdsa, Pair, Public, H160, H256};
+use sp_runtime::{
+	traits::{BlakeTwo256, Hash},
+	Perbill,
+};
 use stake::{InflationInfo, Range};
+use std::convert::TryInto;
 use std::{collections::BTreeMap, str::FromStr};
+use tiny_hderive::bip32::ExtendedPrivKey;
+
+/// Helper function to derive a num_accounts child pairs from mnemonics
+/// Substrate derive function cannot be used because the derivation is different than Ethereum's
+pub fn derive_pairs_from_mnemonic<TPublic: Public>(
```

Comment:
Maybe link to the Substrate code that you're saying doesn't apply here.

Should we call it `derive_bip44_pairs_from_mnemonic`?

Reply:
I like the name